src/Microsoft.DocAsCode.Build.ConceptualDocuments/BuildConceptualDocument.cs

```diff
@@ -84,6 +85,20 @@ public override void Build(FileModel model, IHostService host)
             host.ReportDependency(model, result.Dependency);
         }
 
+        #region ISupportIncrementalBuild Members
+
+        public bool CanIncrementalBuild(FileAndType fileAndType)
```

Comment:
some processor/step always need force-rebuild, for example, it will generate a date time now in some content.